work/v1/types.go

```diff
@@ -125,40 +125,40 @@ type AppliedManifestResourceMeta struct {
 	Namespace string `json:"namespace" protobuf:"bytes,5,opt,name=namespace"`
 }
 
-// ManifestWorkStatus represents the current status of spoke manifest workload
+// ManifestWorkStatus represents the current status of managed manifest workload
```

Comment:
Want to make sure this is intentional, reads a little awkwardly. `managed cluster`?